model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -27,4 +35,29 @@ data class OrtConfiguration(
      * The configuration of the scanner.
      */
     val scanner: ScannerConfiguration? = null
-)
+) {
+    companion object {
+        /**
+         * Load the [OrtConfiguration]. The different sources are used with this priority:
+         *
+         * 1. [Command line arguments][args]
+         * 2. [Configuration file][configFile]
+         * 3. default.conf from resources
+         */
+        fun load(configFile: File? = null, args: Map<String, String> = emptyMap()): OrtConfiguration {
```

Comment:
If `args` as a higher priority than `configFile`, would it makes sense to also make `args` the first parameter to emphasize this?

Reply:
Done.